src/chainparams.cpp

```diff
@@ -482,6 +482,8 @@ class CCustomParams : public CRegTestParams {
 
         consensus.nMinimumChainWork = uint256S(args.GetArg("-con_nminimumchainwork", "0x0"));
         consensus.defaultAssumeValid = uint256S(args.GetArg("-con_defaultassumevalid", "0x00"));
+        // TODO: Embed in genesis block in nTime field with new genesis block type
+        consensus.dynamic_epoch_length = args.GetArg("-dynamic_epoch_length", 10);
```

Comment:
There's no activation defined, nad the value is pre-set:

`src/consensus/params.h:105:    uint32_t dynamic_epoch_length = std::numeric_limits<uint32_t>::max();` to avoid divide by zero errors for when we're not using it.